app.spec.js

```diff
@@ -101,4 +116,40 @@ describe('API', () => {
       expect(response.body).toEqual('Please try adding your note again')
     })
   });
+
```

Comment:
Tests look clean and the expectations are clear. It's hard to argue with the mocked data, either! 🤣